### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by MariaNono**
I removed the unnecessary font-face declarations and files and amended the declarations hat used those values.
I increased the line-height on the footer
and changed the font-weight and font-size on the navbar-item's
I had to remove the class is-size-6 from the __header.njk_ file has this class would only add a value of _font-size: 1rem !important_ to the navbar-item's

### CODE REVIEW

File: src/sass/include/_content.sass [hunk 1]
```diff
@@ -92,7 +92,7 @@ body.protocol
   h2
     color: $tl-blue
     font-family: $family-secondary
-    font-weight: 300
+    font-weight: 500
```
Comment (Smokyish) [inline src/sass/include/_content.sass:95]: Why are we using `500` here while we use `600` on other websites for `h2` and the brand guidelines says 600 for h2?

**Review by Smokyish — CHANGES_REQUESTED** [review]
`body` is missing the `sans-serif` declaration. Many items inherit the family from that, so it is missing in several places. Looks like `$family-primary: 'Inter'` is the place missing that, while the secondary does have it added.

File: src/sass/include/_content.sass (hunk 1 above)
Comment (MariaNono) [reply src/sass/include/_content.sass:95]: My mistake

**Review by Smokyish — CHANGES_REQUESTED** [review]
The `h2` font-weight is still `500` instead of the preferred `600`.

**Review by Smokyish — APPROVED** [review]

**Pull request merged**